src/containers/AccountConnection.jsx

```diff
@@ -141,11 +141,30 @@ class AccountConnection extends Component {
       messages.push({message: 'account config details', params: {folder: account.folderPath}})
     }
 
+    if ([
+      'orange',
+      'orangelivebox',
+      'orangemobile'
+    ].indexOf(account.type) !== -1) {
+      messages.push(
+        {message: 'account config delay', params: {delayInDays: 15}}
+      )
+    }
     this.handleSuccess(account, messages)
   }
 
   handleUpdateSuccess (account) {
-    this.handleSuccess(account, [{message: 'account update success'}])
+    const messages = [{message: 'account update success'}]
+    if ([
+      'orange',
+      'orangelivebox',
+      'orangemobile'
+    ].indexOf(account.type) !== -1) {
+      messages.push(
```

Comment:
The problem with this is that the Notifier does support to have more that 2 parameters apparently so this does not work. Because in case of success, you will get 3 messages : one for "OK", one for the folder path and the 15 days message. Maybe we should join all the messages in ConnectorManagement ?